force-app/main/default/classes/DMLOperations.cls

```diff
@@ -161,13 +188,22 @@ public with sharing class DMLOperations {
 	public static Account upsertAccount(String accountName) {
 		// Query for the Account
 		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
-
+        Account acc = new Account(Description = 'New Account', Name = accountName);
 		// if the Account exists, update the description
-
+        
+            If(!accounts.isEmpty())
```

Comment:
Good job using the List to check and the isEmpty method to see that we have records.